README.md

```diff
@@ -35,13 +35,32 @@ Look for the block comments with text "[BEGIN] Vizbee Integration" and "[END] Vi
 
 ### Styling
 1. The Vizbee SDK components can be customized to match your app's specific theme. Please refer to the below files on how to incorporate them into both light and dark themes.  
-    [vizbee.xml(light)](app/src/main/res/values/vizbee.xml)<br>
-    [vizbee.xml(night)](app/src/main/res/values-night/vizbee.xml)
+   [vizbee.xml(light)](app/src/main/res/values/vizbee.xml)<br>
+   [vizbee.xml(night)](app/src/main/res/values-night/vizbee.xml)
 2. The Vizbee miniPlayer needs to be styled at the app level. Please refer to the files below for guidance.
-    [colors.xml(light)](app/src/main/res/values/colors.xml)<br>
-    [colors.xml(night)](app/src/main/res/values-night/colors.xml)
+   [colors.xml(light)](app/src/main/res/values/colors.xml)<br>
+   [colors.xml(night)](app/src/main/res/values-night/colors.xml)
 
 ## Documentation
 * [Vizbee Android mobile App Developer Guide](https://console.vizbee.tv/app/vzb2000001/develop/guides/android-continuity)
 * [Code Snippets](https://console.vizbee.tv/app/vzb2000001/develop/guides/android-snippets)
 * [Troubleshooting](https://console.vizbee.tv/app/vzb2000001/develop/guides/sender-troubleshooting-snippets)
+
+# Vizbee HomeSSO Demo App
+The Vizbee Android HomeSSO Demo App showcases the integration of the VizbeeHomeSSO Mobile SDK (a wrapper built on top of the Vizbee Continuity SDK) within an Android mobile app.
+
+## Integration Steps for your Android mobile app
+Look for the block comments with text "[BEGIN] Vizbee Integration" and "[END] Vizbee Integration" in the code for an easy understanding of the integration.
+
+### Build Setup
+1. Add the Vizbee repository to your Android mobile app’s root [settings.gradle](settings.gradle).
+2. Add Vizbee HomeSSO SDK dependency to your app module's [build.gradle](/app/build.gradle).
+
+### Code Setup
+1. Copy [VizbeeHomeSSOAdapter](app/src/main/java/tv/vizbee/demo/vizbee/VizbeeHomeSSOAdapter.kt) class, it serves as an implementation of the `IVizbeeHomeSSOAdapter` protocol, providing methods required by the VizbeeHomeSSO SDK during its initialization and operation.
```

Comment:
In Android we call it interface not protocol

Reply:
Done